lib/plugins/plugin/lib/utils.js

```diff
@@ -49,7 +50,20 @@ module.exports = {
 
   getPlugins() {
     const endpoint = 'https://raw.githubusercontent.com/serverless/plugins/master/plugins.json';
-    return fetch(endpoint).then((result) => result.json()).then((json) => json);
+
+    // Use HTTPS Proxy (Optional)
+    const proxy = process.env.proxy
+      || process.env.HTTP_PROXY
+      || process.env.http_proxy
+      || process.env.HTTPS_PROXY
+      || process.env.https_proxy;
+
+    const options = {};
+    if (proxy) {
+      options.agent = new HttpsProxyAgent(url.parse(proxy));
```

Comment:
See the error below:

```
$ serverless plugin  list
 
  Reference Error ----------------------------------------
 
  url is not defined
 
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           11.1.0
     Serverless Version:     1.32.0
```

Reply:
Doh. Fixed. re-install and try again please.